draft-ietf-ace-coap-est-oscore.md

```diff
@@ -286,6 +286,7 @@ It is therefore required that
   * The EST-oscore endpoints support delayed responses
   * The endpoints supports the following CoAP options: OSCORE, Uri-Host, Uri-Path, Uri-Port, Content-Format, Block1, Block2, and Accept.
   * The EST URLs based on https:// are translated to coap://, but with mandatory use of the CoAP OSCORE option.
+    In case DTLS is additionally used, the translation target is the scheme coaps://, instead of coap://.
```

Comment:
If you refer to a "scheme", then you should say:

`In case DTLS is additionally used, the translation target is the scheme "coaps", instead of "coap".`

Reply:
fixed